src/StatefulButton/__snapshots__/StatefulButtontest.test.jsx.snap

```diff
@@ -5,10 +5,9 @@ exports[`StatefulButton renders basic usage 1`] = `
   <button
     aria-disabled={false}
     aria-live="assertive"
-    className="btn pgn__stateful-btn pgn__stateful-btn-state-default btn-primary mr-2"
-    onBlur={[Function]}
```

Comment:
A little confused why the blur/key down handlers would disappear and disabled = false would appear... Doesn't that mean we're not properly passing through our handlers?

Reply:
The old button had noop functions as default prop values.